### ISSUE TITLE
Bundle claypot into executable

### ISSUE THREAD

**Issue by ramnivas** [exograph/exograph#236]
Currently, we deserialize .claypot file which may be produced by an incompatible version and may result in incorrect behavior. And in any case, the current scheme requires shipping two file (clay-server and the claypot file--now that we include *.bundle.js in claypot itself).

Modifying the `.rodata` section of ELF executable (and equivalent Windows format) seems tricky at best. So here is a simpler format:

```
app-specific-executable:
   clay-server executable bytes
   claypot bytes
   <claypot bytes size>
```

So basically, produce clay-server in the usual manner and during `clay build` append what is currently the claypot format along with sufficient information to read the claypot portion.

Then upon loading, load the executable file itself, read the `<claypot bytes size>` portion to find the start of `claypot bytes` and read the `claypot bytes` portion. Then process it like how we do now.
